client/js/components/faq/DpFaqCategoryItem.vue

```diff
@@ -39,9 +39,9 @@
 <script>
 
 export default {
-  name: 'DpFaqItem',
-  components: {
-  },
+  name: 'DpFaqCategoryItem',
+
+  components: {},
```

Comment:
If You already change it, it can be deleted

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3416/commits/49d6bd671d1201a0362a04d91f0765e20db94d6c